.jazzy.json

```diff
@@ -21,10 +21,11 @@
       {
          "name":"Documentation",
          "children":[
-	    "Installation",
-	    "Updating to 3.2.0",
+	          "Installation",
```

Comment:
What happened to these indentations?

Reply:
Honestly I didn't use a proper editor when I created it for the first time, so I had to fix the indentation